src/github_client.py

```
@@ -0,0 +1,121 @@
+              nodes {
+                provider
+                url
+              }
+            }
+          }
+        }
+        """
+        variables = {"username": github_username}
+        
+        try:
+            response = requests.post(
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)